gcc/config/riscv/riscv-c.cc

```diff
@@ -18,6 +18,7 @@ You should have received a copy of the GNU General Public License
 along with GCC; see the file COPYING3.  If not see
 <http://www.gnu.org/licenses/>.  */
 
```

Comment:
This file does not need to be altered.
When you add sub-extendsion in riscv-common.cc , the marco like "__riscv_zbpbo__" will be generated.